### ISSUE TITLE
Video and PDF libraries not working. 

### ISSUE THREAD

**Issue by juliojt** [ruby-processing/JRubyArt#13]
### Environment

This is my result of k9 setup check:
```
JRubyArt version 1.0.5
  PROCESSING_ROOT = E:\processing-3.0.2 Windows
  JRUBY = true
  jruby-complete installed = true
```
jruby -v:

```
jruby 9.0.5.0 (2.2.3) 2016-01-26 7bee00d Java HotSpot(TM) Client VM 25.77-b03 on
 1.8.0_77-b03 +jit [Windows 7-x86]
```

java -version:
```
java version "1.8.0_77"
Java(TM) SE Runtime Environment (build 1.8.0_77-b03)
Java HotSpot(TM) 64-Bit Server VM (build 25.77-b03, mixed mode)
```

### Expected Behavior

Basically, many sketches won't run. All of those that use video libraries (I do have them installed in processing, and I've made copies of them in the sketch directory); and also others that use the processing libraries, like PDF. Examples: 

Ascii_capture:
`https://github.com/ruby-processing/samples4ruby-processing3/blob/master/processing_app/library/video/capture/ascii_capture.rb`

PDF large_page
`https://github.com/ruby-processing/samples4ruby-processing3/blob/master/processing_app/library/pdf/large_page.rb`

### Actual Behavior

```
>k9 --nojruby run ascii_capture.rb

LoadError: no such file to load -- /ascii_capture
                  require at org/jruby/RubyKernel.java:937
                  require at uri:classloader:/META-INF/jruby.home/lib/ruby/stdli
b/rubygems/core_ext/kernel_require.rb:54
        load_ruby_library at C:/jruby-9.0.5.0/lib/ruby/gems/shared/gems/jruby_ar
t-1.0.5/lib/jruby_art/library_loader.rb:44
  block in load_libraries at C:/jruby-9.0.5.0/lib/ruby/gems/shared/gems/jruby_ar
t-1.0.5/lib/jruby_art/library_loader.rb:30
                     each at org/jruby/RubyArray.java:1560
           load_libraries at C:/jruby-9.0.5.0/lib/ruby/gems/shared/gems/jruby_ar
t-1.0.5/lib/jruby_art/library_loader.rb:29
           load_libraries at C:/jruby-9.0.5.0/lib/ruby/gems/shared/gems/jruby_ar
t-1.0.5/lib/jruby_art/app.rb:65
           <class:Sketch> at ascii_capture.rb:9
                   <eval> at ascii_capture.rb:1
              module_eval at org/jruby/RubyModule.java:2711
      load_and_run_sketch at C:/jruby-9.0.5.0/lib/ruby/gems/shared/gems/jruby_ar
t-1.0.5/lib/jruby_art/runners/base.rb:43
                    <top> at C:/jruby-9.0.5.0/lib/ruby/gems/shared/gems/jruby_ar
t-1.0.5/lib/jruby_art/runners/run.rb:5
```

### Other

I've tried copying the libraries to the sketch folder with the appropiate folder structure. It makes no difference. Help?

**Comment by monkstone** [comment]
This is indeed strange, because the program is complaining of missing sketch file in the filesystem root `/` directory (ie not SKETCH root directory), this is most likely a windows path thing and might be fixed by giving an absolute path to the sketch (this should not be required though).  Presumably regular sketches such as JWishy run OK so it must be some interaction between the library loader which futz's with the sketches CLASSPATH so the user doesn't have to.

Without the futz the user would need to add the classpath (of the library) to the command line and being java absolute paths are required so it is a right pain. Back in the day we used to give this advice with ruby-processing https://github.com/jashkenas/ruby-processing/wiki/For-Windows-Users, though I am not sure having  settable K9_HOME (_cf_ RP5_HOME) would fix your problem.

**Comment by monkstone** [comment]
@juliojt it is just possible that recent changes to JRubyArt (and changes to [pdf examples][pdf]) might improves matters.

[pdf]:https://github.com/ruby-processing/JRubyArt-examples/tree/master/processing_app/library/pdf

**Comment by monkstone** [comment]
@juliojt it is not just as simple as needing to quote paths in config file (to cope with illegal spc etc of windows paths)?

**Comment by monkstone** [comment]
No progress or fixed? Someone will re-open if there's a real issue/

**Comment by juliojt** [comment]
Sorry, Martin. I forgot about this email. I haven't verified if quoting
paths helps. I just have too much in my hands these days. Anyways, thanks
for taking your time, and I will try this as soon as I'm able.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DDB4FAA8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Thu, Jun 16, 2016 at 1:03 PM, Martin Prout <notifications@github.com>
wrote:

> Closed #13 <https://github.com/ruby-processing/JRubyArt/issues/13>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ruby-processing/JRubyArt/issues/13#event-694998632>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARjXjfLb06rS5TNZ6GetjmsNtHVcqpoRks5qMZAPgaJpZM4IEveG>
> .
>



-- 
*- Julio*
